### PR TITLE
File VPC library changes

### PR DESCRIPTION
**Pull request by sameshai**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+lint:
+	golangci-lint run
+
+.PHONY: makefmt
+makefmt:
+	gofmt -l -w ${GOFILES}
+
+.PHONY: build
+build:
+	go build -gcflags '-N -l' -o libSample samples/main.go samples/attach_detach.go samples/volume_operations.go
```
Comment (arahamad) [inline Makefile:35]: Can you remove attach_detach related files

File: etc/sample_key_invalid [hunk 2]
```diff
@@ -0,0 +1,26 @@
+MIIEowIBAAKCAQEA4f5wg5l2hKsTeNem/V41fGnJm6gOdrj8ym3rFkEU/wT8RDtn
```
Comment (arahamad) [inline etc/sample_key_invalid:1]: Can we remove these key files

File: etc/sample_key.pub [hunk 3]
```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```
Comment (arahamad) [inline etc/sample_key.pub:1]: Can we remove these key files

File: etc/sample_key [hunk 4]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (arahamad) [inline etc/sample_key:1]: Can we remove these key files

File: etc/libconfig.toml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  g2_token_exchange_endpoint_url = "https://iam.bluemix.net"
+  g2_riaas_endpoint_url = "https://us-south.iaas.cloud.ibm.com"
+  g2_riaas_endpoint_private_url = "https://us-south.iaas.cloud.ibm.com"
+  g2_resource_group_id = "832b3da0130e4a78a2d5a6c62e79a745"
+  g2_api_key = ""
+  provider_type = "g2"
+  vpc_provider_type = "vpc-share"
+  encryption = false
+  iks_token_exchange_endpoint_private_url = "https://us-south.containers.cloud.ibm.com"
+  containers_api_csrf_token = "58a100b5-46b0-41db-9a61-a415747d7993"
```
Comment (arahamad) [inline etc/libconfig.toml:15]: please remove this csrf token, don't checkin such information

**Review by arahamad — CHANGES_REQUESTED** [review]

File: etc/sample_key_invalid (hunk 2 above)
Comment (sameshai) [reply etc/sample_key_invalid]: These are use by token_generator tests.  These are also part of block vpc registery
Below tests will fail.

token_generator_test.go:59: 
        	Error Trace:	token_generator_test.go:59
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:60: 
        	Error Trace:	token_generator_test.go:60
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/no_sample_key", Err:0x2}
        	Test:       	TestTokenGenerator
    token_generator_test.go:63: 
        	Error Trace:	token_generator_test.go:63
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:64: 
        	Error Trace:	token_generator_test.go:64
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/no_sample_key", Err:0x2}
        	Test:       	TestTokenGenerator
    token_generator_test.go:74: 
        	Error Trace:	token_generator_test.go:74
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:75: 
        	Error Trace:	token_generator_test.go:75
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/no_sample_key", Err:0x2}
        	Test:       	TestTokenGenerator
    token_generator_test.go:85: 
        	Error Trace:	token_generator_test.go:85
        	Error:      	Expected value not to be nil.
        	Test:       	TestTokenGenerator
    token_generator_test.go:86: 
        	Error Trace:	token_generator_test.go:86
        	Error:      	Expected nil, but got: &fs.PathError{Op:"open", Path:"/Users/sam3984/go/src/github.com/IBM/ibmcloud-volume-file-vpc/etc/sample_key_invalid", Err:0x2}
        	Test:       	TestTokenGenerator
    provider_test.go:85: {"level":"info","timestamp":"2021-04-27T15:38:39.448+0530","caller":"provider/token_generator.go:79","msg":"Entering getJWTToken","contextCredentials":{"AuthType":"IAM_ACCESS_TOKEN","DefaultAccount":false,"Region":"","IAMAccountID":"test-iks-account","ContextID":""}}

**Review by arahamad — APPROVED** [review]
LGTM

**Comment by arahamad** [comment]
@sameshai , configure travis as well

**Review by akgunjal — APPROVED** [review]
LGTM

**Pull request merged**